.github/workflows/sub_gradle_test_and_build.yml

```diff
@@ -111,6 +111,18 @@ jobs:
         run: |
           docker run --network host -v ${GITHUB_WORKSPACE}/platform/src/test/resources://config stellar/anchor-tests:v0.6.5 --home-domain http://host.docker.internal:8080 --seps 1 10 12 24 31 38 --sep-config //config/stellar-anchor-tests-sep-config.json --verbose
 
+      - name: Upload Artifacts
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: gradle-artifact
+          path: |
+            /etc/hosts
```

Comment:
Do we need `/etc/hosts`?

Reply:
It is just an artifact that we create that may help in debugging though.